### ISSUE TITLE
Persistent UNKNOWN_ERROR after 2-3 minutes of trackOnce location updates

### ISSUE THREAD

**Issue by phil-andrews** [radarlabs/react-native-radar#53]
I need a finer grained GPS location so instead of using background tracking I call trackOnce every 3-5 seconds to get the most recent coordinates of a users device. During testing on a real device while walking around outside, after about 2-3 minutes or 60 updates trackOnce starts throwing `UNKNOWN_ERROR` updates. It's always right around update number 60. 

The error is persistent even through app restarts. I have to delete the app entirely than build and run again to get the error to go aways.

**Steps to reproduce the behavior:**
1. Start clean project using rn typescript template.
2. Follow SDK installation instructions from Radar docs.
3. Install RN package and follow installation instructions.
4. Call Radar from App.tsx in componentDidMount. => 
```    setInterval(() => {
      Radar.trackOnce().then((result: any) => {
        console.log(result)
      }).catch((err: any) => {
        console.log(err)
      });
    }, 3000)```

**Expected behavior**
A normal location update on calling `trackOnce`.

"react-native": "0.57.8",
"react-native-radar": "^2.1.2"
emulator XR
xCode 10

**Comment by phil-andrews** [comment]
Something other than `UNKNOWN ERROR` would be pretty useful in that situation. It's difficult to decipher where that rate limit applies to the `trackOnce` call within the SDK as I wasn't using the REST API. I'm not sure how I would have found it without posting here.

**Comment by coolbrow** [comment]
Yeah, looks like that error string is simply missing from the iOS error codes on the React Native side. We will add this in the next release, thanks for the heads up! 

Since the React Native module uses the native modules under the hood, the rate limits for trackOnce are documented here: https://radar.io/documentation/sdk#ios-foreground

**Comment by phil-andrews** [comment]
Thanks man. I appreciate the quick responses.
